shell.middle/src/main/java/org/metaborg/spoofax/shell/commands/SpoofaxEvaluationCommand.java

```diff
@@ -44,8 +44,9 @@ public String description() {
      * @param args
      *            The String to be parsed and executed.
      */
+    @Override
     public void execute(String... args) {
-        onSuccess.accept("Hai, good job.");
-        onError.accept("Oh no..");
+        onSuccess.accept(new StyledText("Hai, good job."));
+        onError.accept(new StyledText("Oh no.."));
```

Comment:
I think some of this styledtext information should already be present, since output should always be syntax colored.